src/service/ui/src/controllers/user.ts

```diff
@@ -120,9 +136,9 @@ export function updateProfile(req: Request, res: Response) {
 		uri: config.AUTHENTICATION.serviceUrl + '/scim2/Users/' + req.user.id,
 	}
 
-	request(options, (error, response, body) => {
-		if (!error && response.statusCode === 200) {
-			updateUserObject(req, updateProfileObject.CshrUser)
+	request(options, (error: Error, response: Response, body: Body) => {
+		if (!error && response.statusCode == 200) {
```

Comment:
Although, in this case, the `==` should be fine, doesn't the linter enforce triple equals everywhere?